code/__DEFINES/flags.dm

```diff
@@ -24,6 +24,7 @@
 #define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
 
 #define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define TRANSPARENT	8192

```

Comment:
For less common flags, we could have a list of less common flags, normally set to null unless there's a less common flag, in which case it's checked for membership in the list, like species flags are. I think we could use @RemieRichards lazy lists as part of it.

We probably still need (I assume faster to check) binary number flags, but have some of the lesser stuff downgraded?